components/form/validation.md

```diff
@@ -0,0 +1,269 @@
+* [Validate a Complex Model](#validate-a-complex-model)
+* [Fluent Validation](#fluent-validation)
+
+### Validate a Model
+
+````CSHTML
+@* Use the Telerik Edit Form for Blazor to Validate a model *@
+
+@using System.ComponentModel.DataAnnotations
+
+<TelerikForm Model="@person">
+    <FormValidation>
+        <DataAnnotationsValidator></DataAnnotationsValidator>
+    </FormValidation>
+</TelerikForm>
+
+
+@code {
+    public Person person { get; set; } = new Person();
+
+    public class Person
+    {
+        [Required(ErrorMessage = "The First name is required")]
+        public string FirstName { get; set; }
+        [Required(ErrorMessage = "The Last name is required")]
+        public string LastName { get; set; }
+        [Required(ErrorMessage ="Enter the name of the company you work for")]
+        public string CompanyName { get; set; }
+        [Required(ErrorMessage ="Enter your position")]
+        [MaxLength(25, ErrorMessage ="The position can be maximum 25 characters long")]
+        public string Position { get; set; }
+    }
+}
+````
+
+### Validate a Complex Model
+
+You can use the <a href="https://docs.microsoft.com/en-us/aspnet/core/blazor/forms-validation?view=aspnetcore-5.0#nested-models-collection-types-and-complex-types" target="_blank">ObjectGraphDataAnnotationsValidator</a>  inside the Telerik Form for Blazor to validate a nested model.
+
+When using a model with nested objects and fields, specify their `Field` setings as a dot-separate string, do *not* use the `nameof` operator, it does not return the full name of the model.
+
+````CSHTML
+@* Validate a complex model *@
+
+@using System.Dynamic
+@using System.ComponentModel.DataAnnotations
+
+<div class="mt-4" style="margin: 0 auto;">
+    <TelerikForm Model="@MyModel">
+        <FormValidation>
+            <ObjectGraphDataAnnotationsValidator></ObjectGraphDataAnnotationsValidator>
+        </FormValidation>
+        <FormItems>
+            <FormItem Field="StringProperty"></FormItem>
+            <FormItem Field="Child.StringProperty" />
+            <FormItem Field="Child.Child.StringProperty" />
+            <FormItem Field="Child.Child.IntProperty" />
+            <FormItem Field="Child.Child.BoolProperty" />
+            <FormItem Field="Child.Child.DateTimeProperty" />
+        </FormItems>
+    </TelerikForm>
+</div>
+@code {
+    TestModel MyModel { get; set; } = new TestModel();
+
+    abstract class TestBaseClass
+    {
+        [Required(ErrorMessage = "String prop is required")]
+        public string StringProperty { get; set; }
+
+        [Required(ErrorMessage = "Int prop is required")]
+        public int? IntProperty { get; set; }
+
+        [Range(typeof(bool), "true", "true", ErrorMessage = "You must accept.")]
+        public bool BoolProperty { get; set; }
+
+        [Required]
+        public DateTime? DateTimeProperty { get; set; }
+
+        [Required]
+        public ExpandoObject ExpandoProperty { get; set; }
```

Comment:
We could remove the expandoobject as we will include support in v2.

Reply:
Thanks, removed it